js/app.js

```diff
@@ -37,56 +54,49 @@ Question.prototype = {
     }
 };
 
-// Details for each question
-var typeface = new Array(); //GARAMOND
-typeface[0] = new Font(
-    ".garamond",
-    ".garamond",
-    ["Verdana", "Garamond", "Monotype"],
-    "Garamond"
-);
-console.log(typeface);
-
-typeface[1] = new Font(  //HELVETICA
-    ".helvetica",
-    ".helvetica",
-    ["Baskerville", "Sans-serif", "Helvetica"],
-    "Helvetica"
-);
-console.log(typeface);
-
-typeface[2] = new Font( //TIMES
-    ".times",
-    ".times",
-    ["Goudy Old Style", "Serifa", "Times"],
-    "Times"
-);
-console.log(typeface);
-
-typeface[3] = new Font( //VERDANA
-    ".verdana",
-    ".verdana",
-    ["Verdana", "Verdy", "Script"],
-    "Verdana"
-);
-console.log(typeface);
-
-typeface[4] = new Font( //ARIAL
-    ".arial",
-    ".arial",
-    ["Comic Sans", "Trade Gothic", "Arial"],
-    "Arial"
-);
-console.log(typeface);
```

Comment:
I mean something like this:
```javascript
var renderButtons = function(question) {
    $('button.choice').each(function(i) {
        // i'll let you fill this in.
    });
};
```
This will allow us to call `renderButtons` in our click handler function.